templates/components/date.build.hbs

```diff
@@ -1,3 +1,4 @@
+<small v-if="description" class="help-block description">\{{ description }}</small>
```

Comment:
I prefer to put this inside a block element, e.g. `div`. If you want to use `small`, put the `small` tag inside a block element.

This way, the text by default will form a block on top of the input fields and makes it easier to style and adjust layouts for if necessary. In contrary, adding a block container later in the future may cause more problem.

Reply:
@tonytlwu 
In that case we do not even need small tag.